dataset/test/bin_test.cpp

```diff
@@ -132,7 +134,7 @@ class BinTest : public ::testing::TestWithParam<DataArray> {
 
   void expect_near(const DataArrayConstView &a, const DataArrayConstView &b) {
     // "round" last digits for approximate floating point comparison
-    const auto rounding = 100.0 * units::one * buckets::sum(a);
+    const auto rounding = 400.0 * units::one * buckets::sum(a);
```

Comment:
scipp also has `is_approx` https://scipp.github.io/generated/scipp.is_approx.html?highlight=is_approx#scipp.is_approx

Reply:
Not for data arrays, and furthermore the tolerance is absolute, but relative makes more sense here. Should probably improve the tooling to support this.